### PR TITLE
v1 of creators cart front end PR

### PR DESCRIPTION
**Pull request by saisumanthkacham**
pages added- cart ,wishlist, product display, product page, authentication

feat- state management, sorting and filtering

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+
+import './index.css';
+import {ProductPage} from "./pages/productsPage/productPage.jsx"
```
Comment (omkarborude) [inline src/App.jsx]: You can import all components from one File!

File: src/pages/data.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+import faker, { seed } from 'faker';
+
+// import { useEffect,useState } from "react";
+// import axios from 'axios'
```
Comment (omkarborude) [inline src/pages/data.js]: commented Code!

File: src/pages/data.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+//   }
+//   catch(err){
+//     console.log(err)
+//   }
+ 
+// })()
+// }
+
+// export const data= FetchData()
+// console.log("outside the func",data)
```
Comment (omkarborude) [inline src/pages/data.js]: same here!

**Review by omkarborude — APPROVED** [review]
looks good to me!

File: src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+//     console.log("inside the func",data)
+//     }
+//     catch(err){
+//       console.log(err)
+//     }
+   
+// })()
+
+//  },[])
+
```
Comment (kunwar-11) [inline src/App.jsx]: Remove unnecessary comments

**Review by kunwar-11 — APPROVED** [review]
Looks good to me 
Just remove unnecessary comments

File: src/App.jsx (hunk 1 above)
Comment (saisumanthkacham) [reply src/App.jsx]: yeah did that now .

File: src/App.jsx (hunk 4 above)
Comment (saisumanthkacham) [reply src/App.jsx]: actually, I accidentally pushed the code I'm working on so I commented them, anyhow I  fixed it.

File: src/pages/data.js (hunk 3 above)
Comment (saisumanthkacham) [reply src/pages/data.js]: actually, I accidentally pushed the code I'm working on so I commented them, anyhow I  fixed it.

**Pull request merged**